frontend/src/components/EditPolicy.vue

```diff
@@ -35,8 +35,17 @@
             required
           ></v-textarea>
         </ValidationProvider>
+        <v-radio-group
+          v-if="false"
```

Comment:
Leaving the actual UI feature disabled until the permissions/sync logic is implemented for table-based.